internal/util/logging/logram.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// LogRAM implements zap logging entry interception and stores the last 1024 entries in the ring buffer in memory.
+package logging
+
+import (
+	"fmt"
+	"sync"
+
+	"go.uber.org/zap/zapcore"
+)
+
+var LogRAM *logRAM
+
+// logRAM is a storage of log records in memory.
+type logRAM struct {
```

Comment:
When I made a review the first time, I misread many things. One of those things was the algorithm that this logger uses. It's not just "a storage of log records in memory", it specifically uses a circular buffer algorithm. 
Would a different name make more sense? Maybe, just `CircullarBuffer`? It's already inside the `logging` package, so it doesn't need to be called "log" to highlight that it's about logging.

Reply:
> When I made a review the first time, I misread many things. One of those things was the algorithm that this logger uses. It's not just "a storage of log records in memory", it specifically uses a circular buffer algorithm.
> Would a different name make more sense? Maybe, just `CircullarBuffer`? It's already inside the `logging` package, so it doesn't need to be called "log" to highlight that it's about logging.

Don't know.
I indicated that records are stored in memory RAM and not in journals.

Reply:
@fenogentov well, technically, every single variable is stored in memory, but we don't specify it :) What's the difference? What makes this approach different than a default `zap` approach?

Reply:
zap does not store records in memory, but writes to a file or stderr, etc.
well, or almost does not store.
The name doesn’t matter to me at all, I’m ready to change it for anything